templates/emails/base.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>{% load static %}
```

Comment:
You need to note your resources.
Which site/template did you use to build this email ? If someone want to change the email, it may be useful.